packages/iTwinUI-react/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 1.48.2
```

Comment:
I think you forgot to bump package.json version.

Reply:
didnt forget, will bump version when all PRs for 1.48.2 are merged